index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <title></title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!--script src='main.js'></script-->
+</head>
+<body>
+    <nav class="toolbar">
+        <img id="img1" src="image_geometry_1.svg" alt="">
+        <div class="ic_icon">
+          <button id="btn"></button>
+          <button id="btn"></button>
+          <button id="btn"></button>
+        </div>
+    </nav>
+    <section class="headline">
+        <div class="group48">
+            <p id="title"><h1> CharlieGiorgania</h1></p>
+            <p id="text">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</br></br> you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button id="group19">Start collaboration</button>
+        </div>
+        <img id="img2" src="image_geometry_2.png" alt="">
+    </section>
```

Comment:
Your design
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/78251927/2322537d-fb2b-45c3-a351-bc7474764e9a)


Figma
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/78251927/4b6ba89f-29ac-40ae-b705-2a094f1805ab)

Please add the section margins (top, bottom, left, right) defined in the design.

* If it's pretty and works, it will be a wonderful website.